src/powerlaces/boot_cljs_devtools.clj

```diff
@@ -52,16 +57,34 @@
    :middleware ['dirac.nrepl/middleware]})
 
 (boot/deftask cljs-devtools
-  "Add Chrome Devtool enhancements for ClojureScript development."
+  "Add Chrome cljs-devtools enhancements for ClojureScript development."
+  [b ids BUILD_IDS  #{str} "Only inject devtools into these builds (= .cljs.edn files)"]
+  (let [tmp  (boot/tmp-dir!)
+        prev (atom nil)]
+  (comp
+   (boot/with-pre-wrap fileset
+     (doseq [f (relevant-cljs-edn @prev fileset ids)]
+       (let [path     (boot/tmp-path f)
+             in-file  (boot/tmp-file f)
+             out-file (io/file tmp path)]
+         (io/make-parents out-file)
+         (add-cljs-devtools-preload! in-file out-file)))
+     (reset! prev fileset)
+     (-> fileset
+         (boot/add-resource tmp)
+         (boot/commit!))))))
+
+(boot/deftask dirac
+  "Add dirac enhancements for ClojureScript development."
   [b ids        BUILD_IDS  #{str} "Only inject devtools into these builds (= .cljs.edn files)"
    n nrepl-opts NREPL_OPTS edn     "Options passed to boot's `repl` task."
    d dirac-opts DIRAC_OPTS edn     "Options passed to dirac."]
-  (let [tmp (boot/tmp-dir!)
-        prev (atom nil)
-        nrepl-opts (cond-> (merge nrepl-defaults nrepl-opts)
-                     (get-in dirac-opts [:nrepl-server :port]) (assoc :port (get-in dirac-opts [:nrepl-server :port])))
-        dirac-opts (cond-> (or dirac-opts {})
-                     (:port nrepl-opts) (assoc-in [:nrepl-server :port] (:port nrepl-opts)))
+  (let [tmp              (boot/tmp-dir!)
```

Comment:
Ok these guys are aligned so maybe I just need to know how to make it so :smile: I use `aggressive-indent-mode` which just keeps everything formatted following the clojure style guide and probably does not like this :crying_cat_face: